math/matrix/inc/TVectorT.h

```diff
@@ -27,15 +27,15 @@
 template<class Element> class TVectorT : public TObject {
 
 protected:
-   Int_t    fNrows;                // number of rows
-   Int_t    fRowLwb;               // lower bound of the row index
-   Element *fElements;             //[fNrows] elements themselves
+   Int_t    fNrows{0};                // number of rows
```

Comment:
Nice point. I had an analogous discussion with a C++ expert a few days ago. I have no strong opinion here but Herb Sutter seem to lean towards one of the two cases: https://herbsutter.com/2013/05/09/gotw-1-solution/